snips/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 5.1
+- Updated to snips 0.60.1
```

Comment:
Version?
I think we can stay that we make a major release if the user need updates the assistant files otherwise we make minor updates 👍

Reply:
That makes a lot of sense. In this case we don't need to update the assistant files (using assistant version .17 at this time for 0.60.1 and 0.60.8)